### PR TITLE
BOTMETA: Restart ovirt labeling

### PR DESCRIPTION
**Pull request by mwperina**
1. Creates ovirt team
2. Labels following with ovirt label
    - $modules/cloud/misc/ovirt.py
    - $modules/cloud/ovirt/
    - contrib/inventory/ovirt4.py
    - $module_utils/ovirt.py

### CODE REVIEW

**Comment by mwperina** [comment]
@machacekondra

**Comment by mwperina** [comment]
John, could you please take a look?
@gundalow

**Comment by mwperina** [comment]
Fixed issue, John/Dag could please review?

@gundalow @dagwieers

**Review by machacekondra — APPROVED** [review]
shipit

**Review by gundalow — CHANGES_REQUESTED** [review]
Update is good, though on hold as we are fighting some Bot issues at the moment.

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
@mwperina The Bot issue has been resolved, if you could please rebase this PR I'll get it merged.

**Comment by dagwieers** [comment]
I already relabeled all ovirt issues and PRs. However I don't think we really want to have the ovirt and virt WGs separated. There's a lot of overlap and the communities are quite small.

Do we want to break up one small community into 2 smaller communities in this early stage ?

**Comment by mwperina** [comment]
We really need to have all oVirt related issue labelled by unique label, because that's how we identify them and now manually, but in near future automatically, syncs with Jira tasks, where we manage our development process.
We also don't have enough resources to contribute to virt or proxmox modules, so it doesn't make any sense to have single team for all those modules. So I would really appreciate if oVirt modules issues were labeled with ovirt label.
Thanks.

**Comment by dagwieers** [comment]
Ok, let's see how the oVirt WG will evolve.

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -135,12 +135,16 @@ files:
   $modules/cloud/linode/: $team_linode
   $modules/cloud/lxd/: hnakamur
   $modules/cloud/memset/: glitchcrab
-  $modules/cloud/misc/ovirt.py: &virt
+  $modules/cloud/misc/ovirt.py: &ovirt
     ignored: skvidal vincentvdk
```
Comment (dagwieers) [inline .github/BOTMETA.yml]: This should probably not be retained ?

File: .github/BOTMETA.yml [hunk 2]
```diff
@@ -135,12 +135,16 @@ files:
...
-    keywords: [ kvm, libvirt, ovirt, proxmox, qemu, RHEV, RHV ]
+    keywords: [ oVirt, RHV, RHEV ]
+    labels: [ cloud, ovirt ]
+    maintainers: $team_ovirt
+  $modules/cloud/misc/proxmox.py: &virt
+    ignored: skvidal vincentvdk
+    keywords: [ kvm, libvirt, proxmox, qemu ]
     labels: [ cloud, virt ]
     maintainers: $team_virt
-  $modules/cloud/misc/proxmox.py: *virt
```
Comment (dagwieers) [inline .github/BOTMETA.yml:143]: Why is this removed ? Please leave it in.

**Review by dagwieers — CHANGES_REQUESTED** [review]

File: .github/BOTMETA.yml (hunk 1 above)
Comment (mwperina) [reply .github/BOTMETA.yml]: I left it, because it was there in virt, but OK I can remove

File: .github/BOTMETA.yml (hunk 2 above)
Comment (mwperina) [reply .github/BOTMETA.yml:143]: It's still there, see line 143 in my changes. But because in line 138 I've replaced "&virt" with "&ovirt", I have to define "&virt" here. Or am I missing something?

Comment (dagwieers) [reply .github/BOTMETA.yml:143]: Oh right, sorry for that :-)

**Comment by mwperina** [comment]
So can we get that merged?

@dagwieers @gundalow

**Review by gundalow — APPROVED** [review]

**Comment by machacekondra** [comment]
shipit
